src/libinjection_sqli.c

```diff
@@ -14,11 +14,15 @@
 #include <assert.h>
 #include <stddef.h>
 
-#include "version.h"
 #include "libinjection.h"
 #include "libinjection_sqli.h"
 #include "libinjection_sqli_data.h"
 
+// make clang analyzer happy by defining a dummy version
```

Comment:
Should we guard this with `__clang_analyzer__`?

Reply:
Yes, makes sense.